packages/tanium/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.16.0"
```

```suggestion
- version: "1.16.1"
```
Looks like CI is failing because of this.